api/src/test/java/org/openmrs/module/registrationcore/api/mpi/openempi/OpenEmpiPatientFetcherTest.java

```diff
@@ -111,6 +112,7 @@ private void mockResTemplate() {
     }
 
     @Test
+    @Ignore("This functionality is currently not used and not implemented")
```

Comment:
why are these test ignored?

Reply:
I missed to ignore them in the previous PRs. This logic relevant to OpenEMPI Connector is no longer used. The code that fails this test is [here](https://github.com/IsantePlus/openmrs-module-registrationcore/blob/master/api/src/main/java/org/openmrs/module/registrationcore/api/mpi/openempi/OpenEmpiPatientFetcher.java#L48)